medicines/web/components/search-results/index.tsx

```diff
@@ -165,26 +165,26 @@ const SearchResults = (props: { drugs: IDocument[]; lastSearch: string }) => (
               <p className="icon">{drug.docType.toUpperCase()}</p>
             </div>
             <div className="right">
-              <a href={drug.url}>
-                <p className="drug-name">
+              <p className="drug-name">
```

Comment:
Please, if this a title of a result item I suggest to change `<p>` for a heading tag, e.g `<h3>`

Reply:
Resolved

Reply:
Agreed, we should raise an issue to review semantic use of HTML but this is out of scope of this bugfix.